### PR TITLE
Update intro to remove "alpha".

### PR DESCRIPTION
**Pull request by triclambert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ __DC/OS SDK__ is a collection of tools, libraries, and documentation for easy in
 
 ![Status](https://img.shields.io/badge/Status-Alpha-BF97F0.svg?style=flat-square)
```
Comment (elezar) [inline README.md]: I think the status shield should also be changed?

Comment (gabrielhartmann) [reply README.md]: Yes.  That's what I want to remove.  How about `beta` @triclambert ?  Beta says "progressing toward a 1.0 release" to me.  Either that or we drop the shield and say:

```
DC/OS SDK is currently in beta.  It is progressing toward a 1.0 release, but is still under heavy development.
```

Comment (triclambert) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -12,9 +12,9 @@
 ---
 __DC/OS SDK__ is a collection of tools, libraries, and documentation for easy integration and automation of stateful services, such as databases, message brokers, and caching services.
 
-![Status](https://img.shields.io/badge/Status-Alpha-BF97F0.svg?style=flat-square)
+![Status](https://img.shields.io/badge/Status-Beta-BF97F0.svg?style=flat-square)
 
-DC/OS SDK is currently in alpha stage: it can run services, but APIs change regularly, and features are under active development.
+DC/OS SDK is progressing toward a 1.0 release, but as such is still under heavy development.
```
Comment (gabrielhartmann) [inline README.md]: Is "as such" needed?

**Pull request merged**